libcudacxx/include/cuda/std/detail/libcxx/include/__new/allocate.h

```diff
@@ -62,7 +62,7 @@ inline _LIBCUDACXX_INLINE_VISIBILITY constexpr bool __is_overaligned_for_new(siz
 template <class... _Args>
 _LIBCUDACXX_INLINE_VISIBILITY void* __libcpp_operator_new(_Args... __args)
 {
-#if __has_builtin(__builtin_operator_new) && __has_builtin(__builtin_operator_delete)
+#if 0 && __has_builtin(__builtin_operator_new) && __has_builtin(__builtin_operator_delete)
```

Comment:
What's the actual thing that makes us disable this? I'd rather have that spelled out, even if it's currently universal, than dropping in `false`. Or at least a comment as to why this is disabled would be appreciated.